service/src/main/resources/templates/index.html

```diff
@@ -0,0 +1,154 @@
+      background: #fafafa;
+    }
+
+    /* make the schema display full-width */
+    .swagger-ui .model-example .model-box {
+      display: block;
+    }
+
+    /* these take a lot of vertical space by default */
+    .swagger-ui div.info {
+      margin: 25px 0;
+    }
+
+    .swagger-ui .opblock .renderedMarkdown p {
+      margin: 0;
+      font-size: 14px;
+      line-height: 1.2;
+    }
+
+    /* Fix up header text styling */
+    .swagger-ui details {
+      margin-bottom: 20px;
+    }
+    .swagger-ui details summary {
+      cursor: pointer;
+    }
+
+    /* Support classes for header pinning */
+    .swagger-ui .scheme-container {
+      position: relative;
+    }
+    .swagger-ui .scheme-container.pinned {
+      position: fixed;
+      top: 0;
+      right: 0;
+      left: 0;
+      z-index: 100;
+    }
+  </style>
+</head>
+
+<body>
+<div id="swagger-ui"></div>
+<script src="/webjars/swagger-ui-dist/swagger-ui-bundle.js"> </script>
+<script src="/webjars/swagger-ui-dist/swagger-ui-standalone-preset.js"> </script>
+<script th:inline="javascript">
+
+   // Adds support for pinning the auth bar when the user scrolls down far enough to hide the bar
+    const pinLoginPlugin = function(system) {
+      return {
+        afterLoad: function(system) {
+          var offsetY;
+          var authBar;
+          document.addEventListener('scroll', function() {
+            if (offsetY === undefined) {
+              // Note: the auth bar is not a React component, so we can't use the standard plugin approach to modify
+              var authBars = document.getElementsByClassName('scheme-container');
+              if (authBars.length > 0) {
+                authBar = authBars[0];
+                offsetY = authBar.offsetTop;
+              }
+            }
+            if (window.scrollY > offsetY) {
+              authBar.classList.add('pinned');
+            } else {
+              authBar.classList.remove('pinned');
+            }
+          });
+        }
+      }
+    }
+
+    // Removes the online validation since some code gen bugs cause us to have some specs that cause the validator to squawk
+  const clearValidator = function(system) {
+    return {
+      components: {
+        onlineValidatorBadge: function() { return null; },
+      }
+    };
+  }
+
+  window.onload = function() {
+    // Begin Swagger UI call region
+    const ui = SwaggerUIBundle({
+      url: 'openapi.yml',
+      dom_id: '#swagger-ui',
+      deepLinking: true,
+      presets: [
+        SwaggerUIBundle.presets.apis,
+        SwaggerUIStandalonePreset
+      ],
+      plugins: [
+        SwaggerUIBundle.plugins.DownloadUrl,
+        pinLoginPlugin,
+        clearValidator
+      ],
+      layout: 'StandaloneLayout',
+      displayOperationId: true,
+      docExpansion: 'none',
+      defaultModelsExpandDepth: -1, // hide the huge list of schemas under the routes
+      defaultModelExpandDepth: 2, // affects the schema shown for a request or response
```

Comment:
This HTML has changes which may be useful for TDR but aren't necessary for catalog. In the case of catalog, there's no need to hide the list of schemas since they're not very large (also it's collapsed by default).

The HTML is also different from the default as it loads the page with the API groups (`datasets` and `public`) collapsed. This is fine for TDR, which has many API groups and many APIs, but isn't necessary for catalog as it doesn't yet have more than a page of APIs. Can you revert the TDR specific changes, or fix the code so that it loads the way it did before?